### DISCUSSION TITLE
Pluto TV images server

### DISCUSSION THREAD

**Discussion by jkemp814** [iptv-org/community#228] [Show and tell]
Apparently, Pluto TV images server either changed, or is down for the last few days. 

Firefox ERROR:
Hmm. We’re having trouble finding that site.
We can’t connect to the server at images.pluto.tv.

I first noticed it when clicking on a logo link to verify which logo it was. I started getting that error last Thursday. I flush the cache in Kodi and none of the logos loaded. Even some channels higher up in the playlist that were using Pluto images fail to load.

**Comment by BellezaEmporium** [comment]
https://images.pluto.tv/channels/5f8ed0f17564a300082b676a/colorLogoSVG.svg?auto=format&fit=crop&fill=blur&ixlib=react-9.0.3&h=96&w=150&q=75&dpr=1

from what I see on the website, they've changed them all.

**Reply by jkemp814** [reply]
Wow! We have about 996 links, not all have logo links, but I would image 90% do. I will do a regex search, to see just how many have logos with bad links.

**Reply by BellezaEmporium** [reply]
Sure, now those are in the svg format. I don't know if all players support that image type.

Regex would be easy, because we just need to replace the known links with this type of link above. (you can also play with the height and width if you want)

**Reply by jkemp814** [reply]
A lot of the images on Wikipedia are now .svg. Been using GIMP to convert them, because I wasn't sure if the players would handle them. As you mention.
